src/components/text-editor/text-editor.component.tsx

```diff
@@ -143,9 +143,18 @@ export const TextEditor = React.forwardRef<Editor, TextEditorProps>(
       label: labelText,
     });
 
+    const [characterCountAriaLive, setCharacterCountAriaLive] = useState<
+      "off" | "polite"
+    >("off");
+
+    useEffect(() => {
+      setCharacterCountAriaLive(isFocused ? "polite" : "off");
+    }, [isFocused]);
+
     const [characterCount, visuallyHiddenHintId] = useCharacterCount(
       getContent(value).getPlainText(""),
-      characterLimit
+      characterLimit,
+      characterCountAriaLive
```

Comment:
**suggestion**: instead of the `useEffect` and `useState` etc. could we just do a ternary here based on the `isFocused` prop? E.g `isFocused ? "polite" : "off"`

Reply:
I did this just for consistency really. I figured it would be easier to understand it this way as it'll be pretty much the same block of code across the various components that require this functionality. I can make the change though 👍🏻